src/main/kotlin/org/wa/rceditor/application/Styles.kt

```diff
@@ -1,14 +1,50 @@
 package org.wa.rceditor.application
 
+import de.jensd.fx.glyphs.fontawesome.FontAwesomeIcon
+import de.jensd.fx.glyphs.fontawesome.FontAwesomeIconView
+import javafx.geometry.Pos
+import javafx.scene.paint.Color
 import javafx.scene.text.FontWeight
-import tornadofx.Stylesheet
-import tornadofx.box
-import tornadofx.cssclass
-import tornadofx.px
+import tornadofx.*
 
 class Styles : Stylesheet() {
     companion object {
+        val promptColor by cssproperty<Color>("-fx-prompt-text-fill")
+
         val heading by cssclass()
+        val prompt by cssclass()
+        val itemRoot by cssclass()
+        val closeIcon by cssclass()
+        val contentLabel by cssid()
+        val addItemRoot by cssclass()
+        val boldLabel by cssclass()
+
+        val menuIcon by cssclass()
+
+        fun closeIcon() = FontAwesomeIconView(FontAwesomeIcon.CLOSE).apply {
```

Comment:
Is it accepted practice to have functions within the style sheet? Not sure I've seen that in TornadoFX before. If you want to group together the iconography, maybe create an `IconsProvider` class?

Reply:
@mbr4477 I took this example from here https://github.com/edvin/todomvc/blob/master/src/main/kotlin/todomvcfx/tornadofx/app/Styles.kt

Reply:
Okay, that's from the Edvin himself, so I'd stick with what you have then!